{
	"title": "CVE-2024-27133",
	"cve": "CVE-2024-27133",
	"notes": [
		{
			"type": "Description",
			"text": "Insufficient sanitization in MLflow leads to XSS when running a recipe that uses an untrusted dataset. This issue leads to a client-side RCE when running the recipe in Jupyter Notebook. The vulnerability stems from lack of sanitization over dataset table fields."
		},
		{
			"title": "Published",
			"type": "Other",
			"text": "2024-02-23"
		},
		{
			"title": "Modified",
			"type": "Other",
			"text": "2024-02-23"
		}
	],
	"references": [
		{
			"url": "https://github.com/mlflow/mlflow/pull/10893",
			"description": "MISC:https://github.com/mlflow/mlflow/pull/10893"
		},
		{
			"url": "https://research.jfrog.com/vulnerabilities/mlflow-untrusted-dataset-xss-jfsa-2024-000631932/",
			"description": "MISC:https://research.jfrog.com/vulnerabilities/mlflow-untrusted-dataset-xss-jfsa-2024-000631932/"
		}
	]
}
